features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -839,6 +839,21 @@
                 {% endif %}
             </FAPI>
             {% endif %}
+
+            <TLSClientCertificateBoundAccessTokensSupported>{{oauth.oidc.tls_client_certificate_bound_access_tokens_supported}}</TLSClientCertificateBoundAccessTokensSupported>
+
+            <SupportedTokenEndpointAuthMethods>
+                {% for authmethod in oauth.oidc.token_endpoint.auth_methods %}
+                    <SupportedTokenEndpointAuthMethod>{{authmethod}}</SupportedTokenEndpointAuthMethod>
+                {% endfor %}
+            </SupportedTokenEndpointAuthMethods>
```

Comment:
These two should not be server configs but rather resolved from the components running in the system. 
IIRC we don't have any configs for adding the actual components to the system either.

Reply:
Fixed in 9078393eebc9a741942a9a4cd60f477abf103517